modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CLibcurlClientCodegen.java

```diff
@@ -617,11 +617,17 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
         } else {
             setProjectName(defaultProjectName);
         }
+        System.out.println("michele was here preprocessOpenAPI "+projectName);
         additionalProperties.put(PROJECT_NAME, projectName);
+        System.out.println("michele was here preprocessOpenAPI after "+projectName);
     }
 
     public void setProjectName(String projectName) {
-        this.projectName = underscore(projectName.toLowerCase(Locale.ROOT));
+        System.out.println("michele was here setProjectName this1 "+this.projectName);
+        System.out.println("michele was here setProjectName no1 "+projectName);
+        this.projectName = underscore(sanitizeName(projectName.toLowerCase(Locale.ROOT)));
+        System.out.println("michele was here setProjectName this2 "+this.projectName);
+        System.out.println("michele was here setProjectName no2 "+projectName);
```

Comment:
@michelealbano Shall we remove these `System.out.println`?

Reply:
OUCH! I changed the .java to sanitize the input and I forgot to remove them!

Reply:
Done